study-builder/studies/testboston/study-base.conf

```diff
@@ -0,0 +1,57 @@
+{
+  "tenant": {
+    "domain": ${auth0.domain},
+    "mgmtClientId": ${auth0.mgmtClientId},
+    "mgmtSecret": ${auth0.mgmtClientSecret}
+  },
+
+  "umbrella": {
+    "name": "TestBoston",
+    "guid": ${id.study}
+  },
+
+  "study": {
+    "name": "TestBoston",
+    "guid": ${id.study},
+    "studyEmail": ${contact.email},
+    "baseWebUrl": ${baseWebUrl},
+    "irbPassword": ${irbPassword},
+    "recaptchaSiteKey": ${recaptchaSiteKey},
+    "plusCodePrecision": null,
+    "shareParticipantLocation": false,
+  },
+
+  "clients": [
+    {
+      "id": ${auth0.appClientId},
+      "secret": ${auth0.appClientSecret},
+      "passwordRedirectUrl": ${passwordRedirectUrl}
+    },
+    {
+      "id": ${auth0.crcClientId},
+      "secret": ${auth0.crcClientSecret},
+      "passwordRedirectUrl": null
+    },
+  ],
+
+  "adminUser": {
+    "guid": "TESTBOSTONADMINUSER1"
+  },
+
+  "supportedLanguages": [
```

Comment:
I wanted to be able to have languages in the skeleton, so that the PIs can see the different languages on the frontend.